okhttp/src/main/java/okhttp3/HttpUrl.java

```diff
@@ -842,6 +842,19 @@ public String fragment() {
   }
 
   /**
+   * Returns the redacted URL that would only show the domain name.
+   */
+  public String redactedUrl() {
+    Builder builder = newBuilder("/...");
+    if (builder != null) {
```

Comment:
I realise that builder will never be null because the url will have to be valid in order to create a HttpUrl out of its builder.

Fixed